components/Toastr/toastr.html

```diff
@@ -0,0 +1,329 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Vrushabh Dhatrak" />
+    <meta
+      name="description"
+      content="Toastr is a very nice component, and you can show success, error, warning & Informative messages."
+    />
+    <link rel="stylesheet" href="./../../styles/main.css" />
+    <script src="./../../script/header.js" defer></script>
+    <script src="./../../script/navigation.js" defer></script>
+    <script src="./../../script/toast.js" defer></script>
+    <script src="https://code.iconify.design/2/2.1.1/iconify.min.js"></script>
+    <title>Getting Start</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="header_container">
+        <header class="sticky">
+          <img
+            class="img_logo"
+            src="./../../assets/image section/logo.png"
+            alt="header_logo"
```

Comment:
alt message should be human readable.
& instead of header logo you can put your site's name or write 'Header'.

Reply:
done